tests/traits/BoostJsonTest.cpp

```diff
@@ -83,6 +83,19 @@ TEST(BoostJsonTest, VerifyTokenExpirationValid) {
 	verify.verify(decoded_token);
 }
 
+TEST(BoostJsonTest, VerifyTokenExpirationInValid) {
```

Comment:
This needs a better name

Reply:
We usually have an "expired" version of this test but with this API, I do not think that makes sense 

https://github.com/Thalhammer/jwt-cpp/blob/953aab6fe47be9e92fcbf5fb8418dee0d682e7ca/tests/traits/TraitsTest.cpp.mustache#L92C15-L92C15